src/Accounts/Accounts/Rest/Utils.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Azure.Commands.Profile.Rest
+{
+    public class Utils
```

Comment:
Please merge it into InvokeAzRestCommand because it is used by it for the time being.